src/extension.ts

```diff
@@ -19,8 +19,10 @@ import * as clipboard from 'clipboardy';
 
 // Internal dependencies
 import { host } from './host';
+import { loadConfigMapData } from './configMap'
```

Comment:
We do an `import *` of this on line 25 so do we need to promote this one function to unqualified?

Reply:
fixed.